src/common/lib/client/realtime.ts

```diff
@@ -161,6 +161,19 @@ class Channels extends EventEmitter {
     return channel;
   }
 
+  getDerived(name: string, deriveOptions: API.Types.DeriveOptions, channelOptions?: ChannelOptions) {
+    name = String(name);
```

Comment:
I don't think this is necessary. If anything it would be better to throw an error if the value passed isn't already a string